src/Api/Models/Request/Organizations/OrganizationSmSubscriptionUpdateRequestModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Core.Models.Business;
+
+namespace Bit.Api.Models.Request.Organizations;
+
+public class OrganizationSmSubscriptionUpdateRequestModel
```

Comment:
Only orgs can use SM, so we don't need to prefix with `Organization`. And it's better to avoid abbreviations:

`SecretsManagerSubscriptionUpdateRequestModel`.

Reply:
thanks